docs/graphs/graph-queries/traversal-queries/traversal-examples.md

```diff
@@ -3,179 +3,253 @@ sidebar_position: 40
 title: Traversal Examples
 ---
 
-This page has several examples of graph traversals.
+This page presents several examples of graph traversals, starting with a very simple, no-code explanation and ending with examples of how to filter graph results.
 
-## Example Execution
+The graph in the following examples is based on the [traversal graph](../../graph-examples/example-graphs#the-traversal-graph). To follow along, create this example graph and run any of the queries provided in this page.
 
-Let's take a look at a simple example to explain how it works.
+![traversal_graph](/img/graphs/traversal_graph.png)
 
-This is the graph that we are going to traverse:
+## Simple Traversal Overview
 
-![traversal_graph](/img/graphs/traversal_graph.png)
+This section provides an explanation of a simple graph traversal using an example. Each step of the traversal process is broken down as the query is executed.
+
+The query uses the following parameters:
+
+- Start at vertex **A**.
+- Use a `min` depth of 1.
+- Use a `max` depth of 2.
+- Traverse (move) only in `OUTBOUND` direction of edges.
 
-We use the following parameters for our query:
+You can traverse edges and vertexes as a named graph or as a set of collections (anonymous graph, also called collection sets).
```

Comment:
```suggestion
You can traverse edges and vertexes as a named graph or as a set of edge collections (anonymous graph, also called collection sets).
```

Reply:
No here we can only specify edge collections

Reply:
Not really. A longer way of saying this would be:

You can traverse eges and vertexes as a named graph, which looks up your vertices and edges for you, or as a set of edge and document collections, where you specify the collections directly. The latter is called collection sets.

I do not think it would not be accurate to specify just edge collections here.

Reply:
It is like edge collection are the basic thing required for collection sets you can also optionally specify vertex collections too. But the core concept is:
 Instead of GRAPH graphName you may specify a list of edge collections. Vertex collections are determined by the edges in the edge collections.

Reply:
I think that is covered in the syntax page.